e2e/webpack_devserver_esm/lib/mylib/index.js

```diff
@@ -0,0 +1,5 @@
+const packageJson = require('./package.json')
```

Comment:
esm imports?

Reply:
Technically the "esm" refers to the webpack config being esm but may as well make it all consistent 👍